### PR TITLE
Job board authentication UI

### PR DESCRIPTION
**Pull request by egiurleo**
This PR implements the UI for job board authentication. This does not include any actual logic for submitting a password.

Screen Shot 2022-01-21 at 5 19 52 PM

### CODE REVIEW

File: app/javascript/components/pages/JobsAuthenticate.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import SharedLayout from 'components/layout/SharedLayout';
+import Logo from 'components/icons/Logo';
+import Button from 'components/Button';
+
+import 'stylesheets/page';
+import 'stylesheets/jobs_authenticate';
+
+const JobsAuthenticate = () => {
```
Comment (egiurleo) [inline app/javascript/components/pages/JobsAuthenticate.jsx:9]: This is the new page! It's pretty straightforward. The plan is to redirect to this page when an unauthenticated person tries to access the job board. If they submit a valid password, they will be redirected to the job board.

File: app/javascript/components/pages/JobsAuthenticate.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                        value={password}
+                        onChange={(e) => setPassword(e.target.value)}
+                    ></input>
+                    <Button
+                        type="secondary"
+                        className="w-20 mt-3"
+                        onClick={() => console.log(`Password is ${password}`)}
+                    >
+                        {/*TODO: implement actual password submission logic*/}
+                        <button onClick={() => console.log(`Password is ${password}`)}>
```
Comment (egiurleo) [inline app/javascript/components/pages/JobsAuthenticate.jsx:30]: This will be replaced with an actual call to submit the password :)

**Review by stephanieminn — APPROVED** [review]
beaut! my only suggestion is changing the button CTA to something like "enter" or "view jobs". to me, log in implies a user account and some might find that confusing. what do you think?

**Comment by egiurleo** [comment]
@stephanieminn Good call, I changed the copy to "View Job Board" which I think is much clearer. I'm gonna merge but please let me know what you think and I can change it in another PR if necessary.

**Pull request merged**